mlrun/projects/project.py

```diff
@@ -3102,14 +3102,15 @@ def set_model_monitoring_credentials(
         access_key: str = None,
         endpoint_store_connection: str = None,
         stream_path: str = None,
+        tsdb_connection: str = None,
```

Comment:
```py
x: str = None
```
This type-hint is inherently inaccurate, as the default value is `None`.
It should be instead:
```py
x: Optional[str] = None
```
This appears in other changes you made, e.g. `apply_monitoring_serving_graph`.

Reply:
updated for the new changes :)